langchain/chat_models/minimax.py

```diff
@@ -0,0 +1,122 @@
+"""Wrapper around Minimax chat models."""
+import logging
+import requests
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional
+from langchain.llms.utils import enforce_stop_tokens
+
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain.chat_models.base import BaseChatModel
+from langchain.llms.minimax import _MinimaxCommon
+from langchain.schema import (
+    AIMessage,
+    BaseMessage,
+    ChatGeneration,
+    ChatResult,
+    HumanMessage,
+    SystemMessage,
+)
+logger = logging.getLogger(__name__)
+
+@dataclass
+class _ChatHistory:
```

Comment:
@Lukalukaxxx what's this for?

Reply:
@baskaryan  Sorry, this was a piece of unused code. I've already removed it. Please kindly review it again.

> @Lukalukaxxx what's this for?